sema/subyt/sources.py

```
@@ -143,12 +153,12 @@
         self._current_iter = None
         self._ix = -1
 
-    def _exitCurrent(self):
+    def _exitCurrent(self, *exc):
```

Ruff (0.8.2)

Missing return type annotation for private function `_exitCurrent`

Add return type annotation: `None`

(ANN202)